index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
+        <nav>
+            <div id="logo">
+                <h1>My Logo</h1>
+            </div>
+            <div>
+                <i class="fa fa-bars fa-lg"></i>
+            </div>
```

Comment:
- [ ] For site navigation (the hamburger button), it should be wrapped with the `nav` tag instead of the `div` tag. This is the usual convention in coding and it improves the user experience on mobile applications 👍 😎